mapping/sustainability.yaml

```
@@ -1209,16 +1209,16 @@
 
         For each meeting:
 
-        1. Publish the meeting invite. Add a document, set its `.documentType` to 'consultationMeetingInvite' and its `.url` to the URL at which the meeting invite is available.
+        1. Publish the meeting invite. Add a document, set its `.documentType` to 'consultationMeetingInvitation' and its `.url` to the URL at which the meeting invite is available.
 
         2. Publish the meeting details. Add a `Meeting` object to the `.social.consultationMeetings` array and set:
 
         - `.id` incrementally
         - `.date` to the date of the meeting
         - `.address` to the address of the meeting
-        - `.participantCount` to the number of people that participated in the meeting
+        - `.numberOfparticipants` to the number of people that participated in the meeting
 
-        3. Publish the meeting minutes. Add a document, set its `.documentType` to 'consultationMeetingMinutes' and its `.url` to the URL at which the meeting minutes are available.
+        3. Publish the meeting minutes. Add a document, set its `.documentType` to 'minutes.consultationMeeting' and its `.url` to the URL at which the meeting minutes are available.
```

Refer to the common operation for adding a project document.

```suggestion
        3. Publish the meeting minutes. [Add a project document](../common.md#add-a-project-document), set its `.documentType` to 'minutes.consultationMeeting'.
```